src/frontend/Komponenter/Personoversikt/utils.ts

```diff
@@ -2,18 +2,27 @@ import { BehandlingResultat, Fagsak } from '../../App/typer/fagsak';
 import { BehandlingStatus } from '../../App/typer/behandlingstatus';
 import { Behandlingstype } from '../../App/typer/behandlingstype';
 
-export function erAlleBehandlingerErFerdigstilt(fagsak: Fagsak) {
+export function kanOppretteRevurdering(fagsak: Fagsak) {
+    const harMinstEnBehandlingSomIkkeErHenlagt = fagsak.behandlinger.some(
+        (behandling) => behandling.resultat !== BehandlingResultat.HENLAGT
+    );
+
+    const alleBehandlingerErFerdigstiltEllerSattPåVent = fagsak.behandlinger.every(
+        (behandling) =>
+            behandling.status === BehandlingStatus.FERDIGSTILT ||
+            (behandling.status === BehandlingStatus.SATT_PÅ_VENT &&
+                behandling.type === Behandlingstype.REVURDERING)
+    );
+
+    const harBehandlingMedTypeFørstegangsbehandlingEllerRevurdering = fagsak.behandlinger.some(
```

Comment:
Er denne sjekken nødvendig? Føles som at den blir verifisert i `harMinstEnBehandlingSomIkkeErHenlagt`

Reply:
Ja - tenkte det selv først, men den kan finne på å returnere true på tom liste.

Reply:
Når kan det skje? 
`.some` returnerer aldri true for en tom liste

Denne sjekken er alltid true, så lenge det finnes noe i lista

Reply:
Du har rett!

Reply:
Ah - nå husker jeg det. Tilbakekreving og klage ...

Reply:
Ah 👍

Reply:
Vi kan ha harMinstEnBehandlingSomIkkeErHenlagt=true, men bare klage/tilbakekreving. Urimelig at dette skulle kunne skje i praksis, men...